quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/cli/command/AbstractStartCommand.java

```diff
@@ -42,6 +44,7 @@ public void run() {
         doBeforeRun();
         CommandLine cmd = spec.commandLine();
         HttpPropertyMappers.validateConfig();
+        HostnameV2PropertyMappers.validateConfig();
```

Comment:
out-of-scope: We really need some abstraction over our property mappers and have the ability to add validators for the whole category mappers. As the validation for particular mappers should not be here (as shown above).

But it's out of scope of this PR - just sth to consider.

Reply:
#27025 remains open to cover this.